rules/cross-platform/execution_python_script_in_cmdline.toml

```
@@ -0,0 +1,45 @@
+[metadata]
+creation_date = "2021/01/13"
+maturity = "production"
```

```suggestion
maturity = "development"
```

See comment below - I think we wait for eql regex